Library/Formula/onioncat.rb

```diff
@@ -0,0 +1,21 @@
+class Onioncat < Formula
+  desc "VPN-adapter that provides location privacy using Tor or I2P"
+  homepage "https://www.onioncat.org"
+  url "https://www.cypherpunk.at/ocat/download/Source/current/onioncat-0.2.2.r569.tar.gz"
```

Comment:
Would you mind setting an explicit `version` here? It's currently reading as `569`. After that I'm happy to merge this.

```ruby
url "https://www.cypherpunk.at/ocat/download/Source/current/onioncat-0.2.2.r569.tar.gz"
version "0.2.2.r569"
```